### ISSUE TITLE
Allow iFrame to allow embedding of videos

### ISSUE THREAD

**Issue by StfBauer** [microsoft-search/pnp-modern-search#2983]
**Is your feature request related to a problem? Please describe.**
Currently, videos cannot be directly shown in the search result because iFrame gets sanitized.

**Describe the solution you'd like**
Should we add a web component to allow iframe embed in the search result directly - preferable for Microsoft Stream. I could raise a pr for that.

**Describe alternatives you've considered**
Allow iframes to be added directly.

**Comment by wobba** [comment]
Makes sense. We have a video player already, but allowing Stream (SharePoint) as a component makes sense.

**Comment by StfBauer** [comment]
Thats what Stream does ;)

**Comment by wobba** [comment]
It's used in preview hover for videos. Don't remember the exact. Or maybe it was v3. Have too look closer once back from mini-vacation.

**Comment by wobba** [comment]
Checked and it was in v3. You could look at the v3 implementation in the v3 branch and bring something similar over - the `pnp-video-card` component.

**Comment by StfBauer** [comment]
The video card got removed, at least from what I see.

**Comment by wobba** [comment]
@StfBauer feel free to add a new video play component :)
